inc/policyremoveapplication.class.php

```diff
@@ -56,14 +56,10 @@ public function __construct(PluginFlyvemdmPolicy $policy) {
     * @return bool
     */
    public function integrityCheck($value, $itemtype, $itemId) {
-      if ($value == '') {
+      if (!$itemId) {
          Session::addMessageAfterRedirect(__('An application ID is required', 'flyvemdm'));
```

Comment:
The application ID of the message is maybe misleading. This is not an ID from the database but a string like "com.vendor.application" . The value is mandatory because the a gent needs  this to determine the app to uninstall.

Reply:
Now the Id is saved on the task and is mandatory.